src/helpers.js

```diff
@@ -57,8 +56,12 @@ function getWindowFromNode(node) {
   } else if (node.window) {
     // node is window
     return node.window
+  } else if (node.then instanceof Function) {
+    throw new Error(
+      `It looks like you passed a Promise object instead of a DOM node. Did you use a "find" search instead of a "get" or "query"?`,
```

Comment:
Could we change this message to be a little more clear with:

```suggestion
      `It looks like you passed a Promise object instead of a DOM node. Did you do something like \`fireEvent.click(screen.findBy...\` when you meant to do \`fireEvent.click(await screen.findBy...\`?`,
```